dev-lang/python/python-3.13.0_rc1_p3.ebuild

```diff
@@ -125,6 +126,16 @@ pkg_pretend() {
 		ewarn "you can reproduce the problem with dev-lang/python[gil,-jit].  Instead,"
 		ewarn "please consider reporting freethreading / JIT problems upstream."
 	fi
+
+	if use kernel_linux && [[ "${MERGE_TYPE}" != "binary" ]]; then
+		if use pgo || use test; then
+			linux-info_pkg_setup
+			require_configured_kernel
+			local CONFIG_CHECK="DNOTIFY" # bug 938662
+			CONFIG_CHECK+=" CROSS_MEMORY_ATTACH" # bug 938589
+			check_extra_config
```

Comment:
Doesn't this test last-built kernel rather than the currently running kernel? We mustn't make such checks obligatory, or stuff with fail even if people are actually running a good kernel.

Reply:
The `require_configured_kernel` above should enforce that.